README.md

```diff
@@ -116,6 +157,34 @@ Katana is meant to give structure to the logic part of your app. When it comes t
   <a href="https://github.com/BendingSpoons/tempura-swift"><img src="https://raw.githubusercontent.com/BendingSpoons/katana-swift/master/.github/Assets/tempura.png" alt="Tempura" width="240" /></a>
   <a href="https://github.com/BendingSpoons/katana-ui-swift"><img src="https://raw.githubusercontent.com/BendingSpoons/katana-swift/master/.github/Assets/katanaUI.png" alt="Katana UI" width="240" /></a>
 </p>
+## Signpost Logger
```

Comment:
For some reason is not formatted correctly

Reply:
Fixed